src/Components/MasterComponnets/PurchaseOverview/PurchaseOverview.tsx

```diff
@@ -0,0 +1,66 @@
+import { Stack, Typography } from "@mui/material";
+import React from "react";
+import OverViewCard from "../OverViewCard/OverViewCard";
+import ProfitIcone from "../../../Icones/ProfitIcone";
+import CostIcone from "../../../Icones/CostIcone";
+import PurchaseBagIcone from "../../../Icones/PurchaseBagIcone";
+import CancelIcone from "../../../Icones/CancelIcone";
+
+export default function PurchaseOverview() {
+  return (
+    <>
+      <Stack
+        sx={{
+          backgroundColor: "#FFFFFF",
+          height: { xs: "auto", md: "10.2rem" },
+          borderRadius: "0.5rem",
+          padding: { xs: "16px", md: "25px 16px" },
+        }}
+        spacing={2}
+      >
+        <Typography
+          variant="h5"
+          component="h3"
+          sx={{
+            fontSize: { xs: "18px", md: "20px" },
+            fontWeight: "500",
+            lineHeight: { xs: "28px", md: "30px" },
+          }}
+        >
+          Purchase Overview
+        </Typography>
+        <Stack direction={{ xs: "column", md: "row" }} spacing={2}>
+          <OverViewCard
```

Comment:
for dummy data testing, we can make this data inside an array of objects and then map over it. This will be make it easier to make changes when connecting  with real data.